src/platform/landing-pages/facility-sidebar.json

```diff
@@ -0,0 +1,340 @@
+              "expanded": true,
+              "description": null,
+              "url": {
+                "path": "/pittsburgh-health-care/programs"
+              },
+              "links": [
+                {
+                  "label": "Cardiology at VA Pittsburgh",
+                  "expanded": false,
+                  "description": null,
+                  "url": {
+                    "path": "/pittsburgh-health-care/programs/cardiology"
+                  },
+                  "links": []
+                },
+                {
+                  "label": "PIRATE",
+                  "expanded": false,
+                  "description": null,
+                  "url": {
+                    "path": "/pittsburgh-health-care/programs/pirate"
+                  },
+                  "links": []
+                }
+              ]
+            },
+            {
+              "label": "Research",
+              "expanded": true,
+              "description": null,
+              "url": {
+                "path": "/pittsburgh-health-care/research"
+              },
+              "links": [
+                {
+                  "label": "GRECC",
+                  "expanded": false,
+                  "description": null,
+                  "url": {
+                    "path": "/pittsburgh-health-care/research/geriatric-research-education-and-clinical-center-grecc"
+                  },
+                  "links": []
+                },
+                {
+                  "label": "Safety and security",
+                  "expanded": false,
+                  "description": null,
+                  "url": {
+                    "path": "/pittsburgh-health-care/research/safety-security"
+                  },
+                  "links": []
+                },
+                {
+                  "label": "Research professionals",
+                  "expanded": false,
+                  "description": null,
+                  "url": {
+                    "path": "/pittsburgh-health-care/research/research-professionals"
+                  },
+                  "links": []
+                }
+              ]
+            },
+            {
+              "label": "Policies",
+              "expanded": false,
+              "description": null,
+              "url": {
+                "path": "/pittsburgh-health-care/policies"
+              },
+              "links": []
+            },
+            {
+              "label": "Contact us",
+              "expanded": false,
+              "description": null,
+              "url": {
+                "path": "/pittsburgh-health-care/contact-us"
+              },
+              "links": []
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "label": "DAV Vans: transportation for Veterans",
+      "expanded": false,
+      "description": null,
+      "url": {
+        "path": "/syracus-health-care/dav-vans-transportation-for-veterans"
+      },
+      "links": []
+    }
+  ]
+}
```

Comment:
Do we know what this file is for?

Reply:
As a test, I deleted this file and the reference to it. I was still able to run `yarn build` in vets-website, and then `yarn build` and `yarn preview` in content-build and I can stil see the sidebar on facility pages.

Reply:
OK actually, running `yarn watch --env.scaffold` does generate this error with facility-sidebar.json deleted:

```
ERROR in Template execution failed: ReferenceError: facilitySidebar is not defined

ERROR in   ReferenceError: facilitySidebar is not defined

  - dev-template.ejs:37 eval
    [.]/[html-webpack-plugin]/lib/loader.js!./src/platform/landing-pages/dev-template.ejs:37:52
```

... but deleting the sidenav code from `src/platform/landing-pages/dev-template.ejs` resolves that issue.

I don't think we need the side nav widget in vets-website, because (I think) it's only used by the static pages.

Reply:
This file is used for generating the sidebar in one of the landing pages, because there's a Cypress test that will fail if it's not there.

Reply:
Maybe that Cypress test should be moved to content-build?

Reply:
Or at the very least, can we add a comment to the file explaining what it's doing there?

Reply:
Everything under `src/platform/landing-pages` is something needed for the scaffold build, so the purpose should be fairly clear. Maybe it can be improved with a better directory name (`scaffold`?) and/or a `README.md` with some comments on what the files are for.

Reply:
But it is questionable whether that test is valuable if it's just testing something generated from static data like this. It's not going to be kept up to date if there are expected changes to the sidebar.

Reply:
The test that this is needed for is `src/platform/site-wide/side-nav/tests/e2e/sideNav.cypress.spec.js`. That test won't pass in the `content-build` repo's CI because it's testing the `side-nav` which lives in `vets-website`. Eugene brings up a good point that the test may not be valuable if sidebar isn't kept up to date in this file. It looks like that test is mostly just verifying focus on the sidenav?